Source/Menu/Options.cs

```diff
@@ -541,14 +542,13 @@ void buttonOK_Click(object sender, EventArgs e)
         /// <summary>
         /// Returns user's [width]x[height] if expression is valid and values are sane, else returns previous value of setting.
         /// </summary>
-        private string GetValidWindowSize(ComboBox comboWindowSize)
+        private string GetValidWindowSize(string text)
         {
-            // "1024X780" instead of "1024x780" then "Start" resulted in an immediate return to the Menu with no OpenRailsLog.txt and a baffled user.
-            var sizeArray = comboWindowSize.Text.ToLower().Replace(" ", "").Split('x');
-            if (sizeArray.Count() == 2)
-                if (int.TryParse(sizeArray[0], out int width) && int.TryParse(sizeArray[1], out int height))
-                    if ((100 < width && width < 10000) && (100 < height && height < 100000)) // sanity check
-                        return $"{width}x{height}";
+                var match = Regex.Match(text, @"^\s*([1-9]\d{2,3})\s*[Xx]\s*([1-9]\d{2,3})\s*$");//capturing 2 groups of 3-4digits, separated by X or x, ignoring whitespace in beginning/end and in between
```

Comment:
```suggestion
            var match = Regex.Match(text, @"^\s*([1-9]\d{2,3})\s*[Xx]\s*([1-9]\d{2,3})\s*$");//capturing 2 groups of 3-4digits, separated by X or x, ignoring whitespace in beginning/end and in between
```
Delete extra whitespace here.